ci/builder/README.md

```diff
@@ -41,6 +41,9 @@ your latest changes. You can also run a command other than Bash directly:
 $ bin/ci-builder run stable echo "hello from abroad"
 ```
 
+You can specify the parameter `--detach` to start a detached Docker container.
+You can assign a name to the container using the parameter `--name`.
```

Comment:
One difference is that docker supports `--name=value`, but your script doesn't.

Reply:
Happy to change it if that is considered more appropriate here (I actually had it like this before refactoring the arg parsing).

Reply:
I think it's fine as is.